package.json

```diff
@@ -32,11 +32,14 @@
   },
   "devDependencies": {
     "babel-eslint": "10.1.0",
-    "eslint": "7.0.0",
+    "eslint": ">=6.0.0",
```

Comment:
does this need to be in sync with the peer dependencies, or can we use here the latest ESLint?

Reply:
This was already fixed, perhaps you missed a refresh, good point anyway! 🖖

Reply:
I'm still seeing the 6 or later :thinking: See capture below:

![Screenshot from 2020-05-19 14-15-26](https://user-images.githubusercontent.com/583546/82325253-34be5d80-99db-11ea-8031-8c6d6e7d8053.png)

Reply:
Oh sorry I missed your point totally 🤦‍♂️
Fixed now with 2679afe